docs/en/codebases/mmdet3d.md

```diff
@@ -0,0 +1,21 @@
+## MMDetection3d Support
+
+MMDetection3d is a next-generation platform for general 3D object detection. It is a part of the [OpenMMLab](https://openmmlab.com/) project.
+
+### MMDetection3d installation tutorial
+
+Please refer to [getting_started.md](https://github.com/open-mmlab/mmdetection3d/blob/master/docs/en/getting_started.md) for installation.
+
+### List of MMDetection3d models supported by MMDeploy
+
+|       Model        |         Task         | OnnxRuntime | TensorRT | NCNN  | PPLNN | OpenVINO |                                     Model config                                                         |
+| :----------------: | :------------------: | :---------: | :------: | :---: | :---: | :------: | :------------------------------------------------------------------------------------------------------: |
+|    PointPillars    |   VoxelDetection     |      N      |    Y     |   N   |   N   |    N     |     [config](https://github.com/open-mmlab/mmdetection3d/blob/master/configs/pointpillars)     |
```

Comment:
You should mention in doc that pre-processing part is excluded when convertion to ONNX.